### PR TITLE
[WIP] High risk map wip

### PR DESCRIPTION
**Pull request by yshing**

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -255,6 +255,25 @@ exports.onCreatePage = async ({ page, actions }) => {
   })
 }
 
+exports.onCreateWebpackConfig = ({ stage, loaders, actions }) => {
```
Comment (nandiheath) [inline gatsby-node.js:256]: what is the purpose of this method?

File: src/components/highRiskMap.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import React, { Component } from "react"
+
+import ReactDom from "react-dom"
+import PropTypes from "prop-types"
+import L from "leaflet"
+import "leaflet.markercluster"
+import "leaflet.markercluster/dist/MarkerCluster.css"
+import "leaflet.markercluster/dist/MarkerCluster.Default.css"
+import "leaflet/dist/leaflet.css"
+// import mapMarker from "./icons/map-marker.png"
```
Comment (nandiheath) [inline src/components/highRiskMap.js:10]: please remove commented codes if they are not using anymore

File: src/components/highRiskMap.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+// const icons = {
+//   defaultMarker: L.icon({
+//     iconUrl: mapMarker,
+//     iconSize: [30, 30],
+//     iconAnchor: [15, 30],
+//     popupAnchor: [0, -30],
+//   }),
+// }
+const limit = 1.5
+export default class highRiskMap extends Component {
```
Comment (nandiheath) [inline src/components/highRiskMap.js:21]: in the future please consider using functional components with react hook for consistency 😉

File: src/components/highRiskMap.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+      activeDataPoint: {},
+    }
+  }
+
+  dataPointToMarker = highRiskLocation => {
+    const { lat, lng } = highRiskLocation
+    // TODO: Different visual for different type of data.
+    // const marker = L.marker
+    const marker = L.circleMarker
+    return marker([+lat, +lng], {
```
Comment (nandiheath) [inline src/components/highRiskMap.js:53]: 👍

File: src/pages/high-risk.js [hunk 5]
```diff
@@ -295,15 +340,15 @@ const HighRiskPage = ({ data, pageContext }) => {
...
+              setFilters(selectedArray || [])
             }}
           />
           <InfiniteScroll
             list={filteredLocations}
             step={{ mobile: 20 }}
             onItem={(node, index) => (
               <HighRiskCardContainer alignItems="flex-start" key={index}>
-                {item(node, i18n, t)}
+                <Item node={node.node} i18n={i18n} t={t} />
```
Comment (nandiheath) [inline src/pages/high-risk.js]: 👍

**Review by nandiheath — CHANGES_REQUESTED** [review]
LGTM, just come minor changes
will merge tonite anyway

**Comment by yshing** [comment]
Closed in favor of #121

**Pull request closed without merging**